config.js

```diff
@@ -55,7 +55,17 @@ config.lwm2m = {
         }
     ],
     writeFormat: 'application-vnd-oma-lwm2m/text',
-    types: [ ]
```

Comment:
config.js has been modified by this PR. Are needed changes due to the upgrade in dependencies? Or they are just used for testing and should be removed before merging?

Reply:
I forgot to remove my local changes! :(

Reply:
Fixed with a67393681c72102b0ecb4826dde04812007079e5